src/main/java/com/redhat/cloud/custompolicies/app/rest/utils/PagingUtils.java

```diff
@@ -65,6 +69,28 @@ public static Pager extractPager(@NotNull UriInfo uriInfo) {
             }
         }
 
+        List<String> columns = queryParams.get(QUERY_COLUMN);
+        List<String> directions = queryParams.get(QUERY_DIRECTION);
+        if (columns != null) {
+            for (int i = 0; i < columns.size(); ++i) {
+                String column = columns.get(i);
```

Comment:
I think those need to be sanitized. Not sure. Perhaps we should either reject unknown column names. Or pass something like an index and then internally create the column name via that index (actually in both cases an Enum with column names could come handy)